networkx/generators/classic.py

```diff
@@ -490,3 +511,37 @@ def wheel_graph(n,create_using=None):
         G.add_edge(1,n-1)
     return G
 
+
+# TODO On Python 3, we can define this as
+#
+#     def complete_multipartite_graph(*block_sizes, create_using=None):
+#
+def complete_multipartite_graph(*block_sizes, **kw):
+    """Returns the complete multipartite graph with the specified block sizes.
```

Comment:
Oh sure, I can get to this tomorrow. This is just how I write my own docstrings; force of habit.